modules/era/sql/era_dyna_sql.sql

```diff
@@ -1,1077 +1,1084 @@
-----------------------------------------------------------------------
+-- --------------------------------------------------------------------
 --                        Dynamis Era Module                        --
-----------------------------------------------------------------------
-----------------------------------------------------------------------
+-- --------------------------------------------------------------------
+LOCK TABLES `mob_droplist`       WRITE, 
+            `mob_spell_lists`    WRITE,
+            `mob_skill_lists`    WRITE,
+            `mob_skills`         WRITE,
+            `mob_groups`         WRITE,
+            `zone_settings`      WRITE;
+
+-- --------------------------------------------------------------------
 --                          Dynamis-Bastok                          --
-----------------------------------------------------------------------
------------------------------------
+-- --------------------------------------------------------------------
+-- ---------------------------------
 --           Droplists           --
------------------------------------
+-- ---------------------------------
 --            Megaboss           --
 DELETE FROM `mob_droplist` WHERE dropid = "2906"; -- Delete Droplist 2906
-INSERT INTO `mob_droplist` VALUES (2906,0,0,1000,749,100); -- Mythril Beastcoin
-INSERT INTO `mob_droplist` VALUES (2906,0,0,1000,748,100); -- Gold Beastcoin
-INSERT INTO `mob_droplist` VALUES (2906,0,0,1000,1474,150); -- Infinity Core
-INSERT INTO `mob_droplist` VALUES (2906,0,0,1000,1455,150); -- Byne Bill 1
-INSERT INTO `mob_droplist` VALUES (2906,0,0,1000,1455,150); -- Byne Bill 2
-INSERT INTO `mob_droplist` VALUES (2906,0,0,1000,1456,150); -- Hundred Byne Bill
+REPLACE INTO `mob_droplist` VALUES (2906,0,0,1000,749,100); -- Mythril Beastcoin
```

Comment:
We are dropping the entire dropid in this case, wouldn't we have to INSERT rather than replace?

Reply:
REPLACE will act as an INSERT if it doesn't find a matching record.  I don't believe there is a primary key defined on this table so doing a delete and then replace/insert would be necessary to avoid duplication, but either way, a replace will work.

The other element was avoiding errors because of content tags.  Without specifying what columns to insert into, it expects all columns to be included.  Depending on whether or not the table update for content tags is run before or after this file, it would spit out a different error.  So, I just ignored the content tag column, and it can be updated afterwards if need be.

Reply:
Sounds good! As long as it doesn't break things we are good to go. 😄